x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/InitialNodeSecurityAutoConfiguration.java

```diff
@@ -211,70 +211,176 @@ private static void outputInformationToConsole(
+                    + cmdOff
+                    + ", using the enrollment token that you generated."
+            );
+        } else if (Strings.isEmpty(nodeEnrollmentToken)) {
+            builder.append(infoBullet + " Configure other nodes to join this cluster:");
+            builder.append(System.lineSeparator());
+            builder.append(bullet + " On this node:");
+            builder.append(System.lineSeparator());
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Create an enrollment token with "
+                    + cmdOn
+                    + "bin/elasticsearch-create-enrollment-token -s node"
+                    + cmdOff
+                    + "."
+            );
+            builder.append(System.lineSeparator());
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Uncomment the "
+                    + boldOnANSI
+                    + "transport.host"
+                    + boldOffANSI
+                    + " setting at the end of "
+                    + boldOnANSI
+                    + "config/elasticsearch.yml"
+                    + boldOffANSI
+                    + "."
+            );
+            builder.append(System.lineSeparator());
+            builder.append("  " + hyphenBullet + " Restart Elasticsearch.");
+            builder.append(System.lineSeparator());
+            builder.append(bullet + " On the other node:");
             builder.append(System.lineSeparator());
-            builder.append(caCertFingerprint);
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Start Elasticsearch on other nodes with "
+                    + cmdOn
+                    + "bin/elasticsearch --enrollment-token <token>"
+                    + cmdOff
+                    + ", using the enrollment token that you generated."
+            );
+        } else {
+            builder.append(infoBullet + " Configure other nodes to join this cluster:");
             builder.append(System.lineSeparator());
+            builder.append(
+                bullet
+                    + " Copy the following enrollment token and start new Elasticsearch nodes with "
+                    + cmdOn
+                    + "bin/elasticsearch --enrollment-token <token>"
+                    + cmdOff
+                    + " (valid for the next 30 minutes):"
+            );
+            builder.append(System.lineSeparator());
+            builder.append("  " + boldOnANSI + nodeEnrollmentToken + boldOffANSI);
+            builder.append(System.lineSeparator());
+            builder.append(System.lineSeparator());
+            builder.append("If you're running in Docker, copy the enrollment token and run:");
+            builder.append(System.lineSeparator());
+            builder.append(cmdOn + "docker run --name <node-name> -p <host-http-port>:9200 --net elastic -it ");
```

Comment:
it's actually simpler, the command need only be
```
docker run --name <node-name> -p <host-http-port>:9200 --net elastic -e "ENROLLMENT_TOKEN=<token>"
```

Reply:
@jkakavas, is passing the Elasticsearch version not required anymore? I thought that you still needed to include `docker.elastic.co/elasticsearch/elasticsearch:<version>`. If so, I'll update the instructions for [enrolling nodes](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/docker.html#_enroll_additional_nodes) in Docker. Either way, I think that `docker.elastic.co/elasticsearch/elasticsearch:` is duplicated unnecessarily in step 2 of that section.

Reply:
> @jkakavas, is passing the Elasticsearch version not required anymore?

It is. Apologies, the section above wasn't meant to be a complete example, just an example of how the enrollment process is passed. I'll revise to make it clear and complete

Reply:
> Either way, I think that docker.elastic.co/elasticsearch/elasticsearch: is duplicated unnecessarily in step 2 of that section.

++

Reply:
Thanks to both of you! I will amend, I haven't tested the docker command.